Kernel/Devices/Storage/NVMe/NVMeQueue.h

```diff
@@ -34,6 +34,11 @@ enum class QueueType {
 class AsyncBlockDeviceRequest;
 
 struct NVMeIO {
+    void set_as_unused()
```

Comment:
Instead of `set_as_unused`, we could rename it to `clear` or `reset`, and also set `end_io_handler` to `nullptr`.
One more question, just setting `request = nullptr` is enough or should we do an explicit `unref`?

Reply:
Setting `request` to nullptr is enough.